Blockzilla/OnboardingViewController.swift

```diff
@@ -0,0 +1,350 @@
+        let stackView = UIStackView(arrangedSubviews: [protectionImageView, protectionTextStackView])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.alignment = .top
+        stackView.distribution = .fill
+        stackView.spacing = 18
+        stackView.accessibilityIdentifier = "OnboardingViewController.protectionStackView"
+        return stackView
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addSubViews()
+        
+    }
+    
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        if UIDevice.current.userInterfaceIdiom == .phone {
+            if UIDevice.current.orientation.isLandscape {
+                mainStackView.snp.remakeConstraints { make in
+                    make.centerX.equalTo(scrollView.snp.centerX)
+                    make.width.equalTo(scrollView.snp.width)
+                    make.top.greaterThanOrEqualToSuperview()
+                    make.bottom.lessThanOrEqualToSuperview()
+                }
+            } else {
+                mainStackView.snp.remakeConstraints { make in
+                    make.centerX.equalTo(scrollView.snp.centerX)
+                    make.centerY.equalTo(scrollView.snp.centerY)
+                    make.width.equalTo(scrollView.snp.width)
+                    make.top.greaterThanOrEqualToSuperview()
+                    make.bottom.lessThanOrEqualToSuperview()
+                }
+            }
+        }
+    }
+    
+    func addSubViews() {
+        
+        view.addSubview(scrollView)
+        scrollView.snp.makeConstraints { make in
+            make.centerX.equalTo(self.view.snp.centerX)
+            make.width.equalTo(self.view.snp.width)
+            make.top.equalTo(self.view.safeAreaLayoutGuide.snp.top)
+            make.bottom.equalTo(self.view.snp.bottom)
+        }
+        
+        scrollView.addSubview(mainStackView)
+        mainStackView.snp.makeConstraints { make in
+            make.centerX.equalTo(scrollView.snp.centerX)
+            make.centerY.equalTo(scrollView.snp.centerY)
+            make.width.equalTo(scrollView.snp.width)
+            make.top.greaterThanOrEqualToSuperview()
+            make.bottom.lessThanOrEqualToSuperview()
+        }
+        
+        view.backgroundColor = .white
+
+        mozillaIconImageView.snp.makeConstraints { $0.width.height.equalTo(60) }
+        
+        incognitoImageView.snp.makeConstraints { $0.width.height.equalTo(20) }
+        historyImageView.snp.makeConstraints { $0.width.height.equalTo(20) }
+        protectionImageView.snp.makeConstraints { $0.width.height.equalTo(20) }
+        
+        startBrowsingButton.snp.makeConstraints { make in
+            make.width.equalTo(232)
+            make.height.equalTo(36)
+        }
```

Comment:
We have a class in the project `UIConstants` where we set values for layout. And then you can access the property like this: `UIConstants.layout.propertyName`. I suggest to add the values like this `make.width.equalTo(232)` there

Reply:
fixed